Regressed by https://github.com/unicode-org/icu/commit/c3fe7e09d844

libcdr_utils.cpp:111:35: error: use of undeclared identifier 'TRUE'
    ucsdet_enableInputFilter(csd, TRUE);
                                  ^

--- libcdr/src/lib/libcdr_utils.cpp.orig	2020-02-02 15:30:44 UTC
+++ libcdr/src/lib/libcdr_utils.cpp
@@ -108,7 +108,7 @@ static unsigned short getEncoding(const unsigned char 
     csd = ucsdet_open(&status);
     if (U_FAILURE(status) || !csd)
       return 0;
-    ucsdet_enableInputFilter(csd, TRUE);
+    ucsdet_enableInputFilter(csd, true);
     ucsdet_setText(csd, (const char *)buffer, bufferLength, &status);
     if (U_FAILURE(status))
       throw libcdr::EncodingException();
